xilem/src/any_view.rs

```diff
@@ -21,8 +21,10 @@ use crate::{Pod, ViewCtx};
 /// Note that `Option` can also be used for conditionally displaying
 /// views in a [`ViewSequence`](crate::ViewSequence).
 // TODO: Mention `Either` when we have implemented that?
-pub type AnyWidgetView<State, Action = ()> =
+pub type BoxedAnyWidgetView<State, Action = ()> =
     Box<dyn AnyView<State, Action, ViewCtx, Pod<DynWidget>> + Send + Sync>;
```

Comment:
Perhaps the most consistent option is:
```suggestion
pub type AnyWidgetView<State, Action = ()> = 
     dyn AnyView<State, Action, ViewCtx, Pod<DynWidget>> + Send + Sync;
```